src/sql/workbench/contrib/queryplan2/browser/media/queryPlan2.css

```diff
@@ -0,0 +1,71 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.qp2-actionbar-container {
+	height: 100%;
+	width: 30px;
+	position: absolute;
+	top: 0px;
+	right: 0px;
+}
+
+.qp2-container {}
+
+.query-plan2-container {
+	width: 100%;
+	height: 500px;
+	position: relative;
+	border-top: 1px solid;
+}
+
+.qp-properties-container {
+	position: absolute;
+	top: 0px;
+	right: 32px;
+	height: 100%;
+	overflow-y: scroll;
+	background-color: #eeeeee;
+	width: 510px;
+}
+
+.prop-table-header {
+	font-weight: bold;
+}
+
+.properties-header {
```

Comment:
this one is still not specific enough. btw, I noticed that you have updated the selector name, you can also achieve what I said by adding more selectors, e.g. `.qp2-container .properties-header`, assuming `qp2-container` is the class name of an ancestor node.